tests/bagatom/test_updateObjectFromXML.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+
+from copy import deepcopy
```

Comment:
Nope you are technically correct.

The reason I put `pytest` at the top is because I like the `import x` to be first, and the `from x import y` to be second.

But we should probably stick to the PEP8 suggestion.